### PR TITLE
Added guard pokemon icon to map label

### PR DESCRIPTION
**Pull request by topher-au**
Does what it says on the box

### CODE REVIEW

**Comment by MRokas** [comment]
What's up with `map.html`? Trying to take credit for this stuff, huh?

**Comment by clbuttic** [comment]
Got the popcorn out.

**Comment by Znuff** [comment]
This PR explains nothing.

**Comment by Kruemmelmonster** [comment]
what a douchebag move ... but on a serious note this is a nice addition

**Comment by DeviantEng** [comment]
I'm sure the `map.html` was am accident in his commit. I have mine customized also, and also added Google Analytics and a AdSense ad to mine.

File: templates/map.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
 		<!-- Header -->
 		<header id="header">
 			<a href="#nav">Options</a>
-			<h1><a href="#">Pokémon Go Map</a></h1>
+			<h1><a href="http://launceston.pokegogps.com/">Pokémon Go Launceston</a></h1>
```
Comment (jnovack) [inline templates/map.html]: Please remove.

File: templates/map.html [hunk 2]
```diff
@@ -105,7 +105,7 @@
 				</div>
 			</div>
 
-			<div class="form-control switch-container">
+			<div class="form-control switch-container" style="display:none;">
```
Comment (jnovack) [inline templates/map.html]: Why is this hidden?

Comment (willgrnr) [inline templates/map.html]: @jnovack probably for the same reason as above, forgot to remove the customizations before submitting a PR.

Comment (MRokas) [inline templates/map.html]: Why even add it in first place?

Comment (jnovack) [inline templates/map.html]: Now @MRokas don't go asking logical questions, it'll only get you in trouble.

@topher-au if you want this PR to be taken seriously:

1. edit the top post to confirm to the PR standards.
2. rebase and squash to 1 commit.

**Comment by topher-au** [comment]
Didn't realize subsequent commits after I submitted the PR would be added to it.

I could fix the issue and resubmit, but with the nastiness from above commenters over a genuine mistake, I'm not really inclined to put in any further effort, so I'm going to go ahead and close the PR.

**Pull request closed without merging**